irispy/iris_tools.py

```diff
@@ -66,18 +73,21 @@
 # Define whether IRIS WCS is 0 or 1 origin based.
 WCS_ORIGIN = 1
 
-def get_iris_response(pre_launch=False, response_file=None, response_version=None,
-                      force_download=False):
+
+def get_iris_response(time_obs=None, pre_launch=False, response_file=None, response_version=None,
```

Comment:
Theres a bug here if someone gives a `time_obs = parse_time('2013-09-03')`. or if its not in an array format. I think you should have this as a check. Also you should make sure that its a time object - currently it will do something with a string which is buggy.